client/my-sites/stats/hooks/use-plan-usage-query.ts

```diff
@@ -32,9 +33,10 @@ export default function usePlanUsageQuery(
 	siteId: number | null
 ): UseQueryResult< PlanUsage, unknown > {
 	return useQuery( {
+		...getDefaultQueryParams< PlanUsage >(),
 		queryKey: [ 'stats', 'usage', siteId ],
 		queryFn: () => queryPlanUsage( siteId ),
 		select: selectPlanUsage,
-		// staleTime: 1000 * 60 * 5, // 5 minutes
+		staleTime: 10 * 1000,
```

Comment:
10s is long enough to reduce server load but short enough for the data to refresh.